src/cli/configuration_builder.js

```diff
@@ -117,6 +117,9 @@ export default class ConfigurationBuilder {
       const parts = compiler.split(':')
       extensions.push(parts[0])
       require(parts[1])
+      if(parts.length > 1) {
+        require.extensions[parts[2]] = () =>  {}
```

Comment:
I don't see any reason for this to live in cucumber-js. There could be some other plugin that allows you to register what you want returned for files with specific extensions. This could then be required in a test helper before requiring your code